js/ui/camera.js

```diff
@@ -29,7 +29,8 @@ const Evented = require('../util/evented');
  *
  * @typedef {Object} AnimationOptions
  * @property {number} duration The animation's duration, measured in milliseconds.
- * @property {Function} easing The animation's easing function.
+ * @property {Function} easing A function taking a time in the range 0..1 and returning a number where 0 is 
+ *                             the initial state and 1 is the final state.
```

Comment:
Eh, I think it was just an overly-cautious red flag instinct -- it's an assumption I could imagine making if I were implementing animations w/ a custom easing function -- but now that I look at `camera.js`, I don't see any obvious cases where we assume it.